src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -425,11 +425,15 @@ else if(mode[0].equals("username")){
           cu.setPatronGroup((PatronGroup)cf.get().convertToPojo(PatronGroup.class) );
         }
         cf = completedLookup.get(PERMISSIONS_INCLUDE);
-        if(cf != null && cf.get().getBody() != null){
+        if(cf != null && cf.get().getBody() != null && !cf.get().getBody().getJsonArray("permissionUsers").isEmpty()) {
```

Comment:
Do we have unit or integration tests on this part of code?

Reply:
added unit test